llvm/lib/Target/NVPTX/NVPTXInstrInfo.td

```diff
@@ -555,6 +555,34 @@ multiclass F2<string OpcStr, SDNode OpNode> {
                            [(set Float32Regs:$dst, (OpNode Float32Regs:$a))]>;
 }
 
+multiclass F2_Support_Half<string OpcStr, SDNode OpNode> {
+   def bf16 :     NVPTXInst<(outs Int16Regs:$dst), (ins Int16Regs:$a),
```

Comment:
Nit: it would be nice to align all NVPTXInst on the same column.

Reply:
Done